recipes/n88util/build.sh

```diff
@@ -0,0 +1,18 @@
+
+# Create build directory
+mkdir build
+cd build
+BUILD_CONFIG=Release
+
+# CMake
+cmake .. \
+	-G "Ninja" \
+  -DCMAKE_BUILD_TYPE=$BUILD_CONFIG \
```

Comment:
indentation ...

Reply:
Again, spaces vs tabs. Sorry about that.